code/game/world.dm

```diff
@@ -43,7 +43,7 @@ GLOBAL_PROTECT(security_mode)
 
 /world/proc/SetupExternalRSC()
 #if (PRELOAD_RSC == 0)
-	GLOB.external_rsc_urls = world.file2list("config/external_rsc_urls.txt","\n")
+	GLOB.external_rsc_urls = world.file2list("[global.config.directory]/external_rsc_urls.txt","\n")

```

Comment:
Why are you making something depend on config existing before config is created?

Did you even bother to think out initialization order or logic with this pr?

Literally 15 lines up you can see the config controller is created after this is called.